admin-manual/installation-setup/installation/install-ubuntu.rst

```diff
@@ -100,8 +100,15 @@ Ubuntu 16.04 (Xenial) installation instructions
 
    .. code:: bash
 
-      sudo wget https://bootstrap.pypa.io/get-pip.py
-      sudo python get-pip.py
+      curl -Ls https://bootstrap.pypa.io/get-pip | sudo python - "pip==9.0.3"
+
+   Notice that we are using ``pip 9.0.3``, this is important if you are using
+   Ubuntu 14.04 (see `issue #1065`_). Ubuntu 16.04 works with more recent
+   versions of ``pip`` so you could run the following instead:
+
+   .. code:: bash
+
+      curl -Ls https://bootstrap.pypa.io/get-pip | sudo python -
```

Comment:
This change is for Ubuntu Trusty section below, this is the Ubuntu Xenial section.

Reply:
Fixed!!! :D